packages/create-svelte/shared/+eslint/.eslintignore

```diff
@@ -1,13 +1,8 @@
-.DS_Store
-node_modules
-/build
-/.svelte-kit
-/package
-.env
-.env.*
-!.env.example
+.output
+.svelte-kit
+.vercel
 
-# Ignore files for PNPM, NPM and YARN
```

Comment:
are we sure eslint ignores these files? how do we know that?

Reply:
JSON and YAML files are not formatted by default.

Plugins have to be installed:

JSON https://github.com/kuceb/eslint-plugin-json-format
YAML https://github.com/ota-meshi/eslint-plugin-yml

And the extensions should be provided in the CLI and configuration.